packages/theming/apple-theme/src/appleColors.ios.ts

```diff
@@ -421,5 +421,10 @@ export function paletteFromAppleColors(isDark: boolean): ThemeColorDefinition {
     buttonDisabledSecondaryContent: fluentApple.buttonTitleDisabled,
     buttonHoveredSecondaryContent: fluentApple.communicationBlue,
     buttonPressedSecondaryContent: fluentApple.communicationBlueTint20,
+
+    checkboxBackground: fluentApple.communicationBlue,
+    checkboxBackgroundDisabled: fluentApple.surfacePrimary,
+    checkboxBorderColor: fluentApple.gray600,
+    checkmarkColor: fluentApple.iconOnAccent,
```

Comment:
Thank you for updating the apple platform as well =) Did you also get these colors from Figma?

Reply:
Yep. Used the matching tokens with the same values as in Figma.